packages/next-codemod/bin/next-codemod.ts

```diff
@@ -68,6 +69,17 @@ program
   )
   .usage('[revision] [options]')
   .option('--verbose', 'Verbose output', false)
-  .action(runUpgrade)
+  .action(async (revision, options) => {
+    try {
+      await runUpgrade(revision, options)
+    } catch (error) {
+      if (!options.verbose && error instanceof BadInput) {
+        console.error(error.message)
+      } else {
+        console.error(error)
+      }
+      process.exit(1)
+    }
+  })
```

Comment:
Usually there's a global handler in these CLI parser that should be used for that. However, I couldn't find any and the `exitOverride` method was ignored.